.github/workflows/ci.yml

```diff
@@ -11,10 +11,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Setup Python
+      - name: Setup Python 3.8
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.8
       - name: Use pip cache
         uses: actions/cache@v2
         with:
```

Comment:
I did notice 2 other 3.7's below this, do they need to be changed as well? under `Build documentation` and `gnomad_qc`

Reply:
Oops. Thanks for catching those.